### ISSUE TITLE
Elliptical arc isn't being transformed correctly when using a negative scale

### ISSUE THREAD

**Issue by alexjlockwood** [svg/svgo#803]
I am seeing some buggy behavior when I import the below SVG.

Original SVG: 

```
<svg xmlns="http://www.w3.org/2000/svg" width="160" height="160" viewBox="0 0 160 160">
  <path fill="#000" 
      transform="translate(160) scale(-1, 1)" 
      d="M 0 160 V 80 A 80 80 0 0 1 80 0 h 80 V 80 A 80 80 0 0 1 80 160 Z"/>
</svg>
```

<img width="1920" alt="screen shot 2017-09-28 at 12 30 07 pm" src="https://user-images.githubusercontent.com/1380747/30978529-cd8551f8-a448-11e7-8fa5-3295810a0349.png">

Optimized SVG:

```
<svg xmlns="http://www.w3.org/2000/svg" width="160" height="160" viewBox="0 0 160 160">
  <path fill="#000" 
      d="M 160 160 V 80 A 80 80 0 0 1 80 0 V 80 A 80 80 0 0 1 80 160 h 80 z"/>
</svg>
```

<img width="1920" alt="screen shot 2017-09-28 at 12 30 03 pm" src="https://user-images.githubusercontent.com/1380747/30978521-c7ba5fac-a448-11e7-93d2-6bd32cf0a612.png">

**Comment by alexjlockwood** [comment]
Not totally sure, but I think the issue might be here:

https://github.com/svg/svgo/blob/master/plugins/_path.js#L265

From what I can tell, the elliptical arc commands are not having their sweep flags toggled correctly.

**Comment by GreLI** [comment]
I guess it's been fixed by #752.
